STAC-24518 Document otel collector with Application Collection (#338)
rancher/stackstate-product-docs: docs/latest/modules/en/pages/setup/otel/languages/node.js/auto-instrumentation-of-lambdas.adoc

Depending on the desired functionality, or based upon factors such as volumes of data being generated by lambdas instrumented in this way, collectors can be set up for batching, tail-sampling, and other pre-processing techniques to reduce the impact on SUSE Observability.

> See this page for xref:/setup/otel/getting-started/getting-started-lambda.adoc[guidance and instruction] on how to set up a batching collector that acts as a security proxy for SUSE Observability.

Reviewer: ```suggestion
For more information on how to set up a batching collector that acts as a security proxy for {stackstate-product-name}, see xref:/setup/otel/getting-started/getting-started-lambda.adoc[Getting started with Lambda]. 
```
